plugins/source/aws/codegen/recipes/ssoadmin.go

```diff
@@ -0,0 +1,22 @@
+package recipes
+
+import (
+	types "github.com/aws/aws-sdk-go-v2/service/ssoadmin/types"
+)
+
+func SSOAdminResources() []*Resource {
+	resources := []*Resource{
+
+		{
+			SubService: "instances",
+			Struct:     &types.InstanceMetadata{},
+		},
+	}
+
+	// set default values
+	for _, r := range resources {
+		r.Service = "ssoadmin"
+		r.Multiplex = `client.ServiceAccountRegionMultiplexer("identitystore")`
```

Comment:
```suggestion
		r.Multiplex = `client.ServiceAccountRegionMultiplexer("sso")`
```

Should this be `sso`? I believe this should match https://github.com/cloudquery/cloudquery/blob/18c7a3224ba898451c1a29395acabc2e4395a40b/plugins/source/aws/client/data/partition_service_region.json#L5219

Reply:
No, actually, we need to make a new entry in that file for the `ssoadmin` service. There are 17 regions that it supports, though it can only be configured in one region at a time. @bbernays confirmed to me on Discord and in the related issue ticket that this will be ok.

Reply:
Thanks @jhudson10x for the added context, makes sense 👍